A/AbstractXsdTypes/Package.toml

```diff
@@ -1,4 +1,4 @@
 name = "AbstractXsdTypes"
-uuid = "1d4aff4b-e088-40cb-b713-ffe4caf1580b"
+uuid = "894546dd-dc3f-42e8-9b69-a7785ccf72be"
```

Comment:
You also need to change the uuid in the top-level Registry.toml file (and move it to the right place)